docs/FlowCollector.md

```diff
@@ -5958,8 +5958,8 @@ the kernel debug filesystem, so the eBPF pod has to run as privileged.
 If the `spec.agent.ebpf.privileged` parameter is not set, an error is reported.<br>
 - `DNSTracking`: enable the DNS tracking feature.<br>
 - `FlowRTT`: enable flow latency (sRTT) extraction in the eBPF agent from TCP traffic.<br>
-- `NetworkEvents`: enable the Network events monitoring feature.  This feature requires mounting
-the kernel debug filesystem, so the eBPF pod has to run as privileged.<br/>
+- `NetworkEvents`: enable the Network events monitoring feature. This feature requires mounting
+the kernel debug filesystem, so the eBPF pod has to run as privileged. It is currently a Developer Preview.<br><br/>
```

Comment:
```suggestion
the kernel debug filesystem, so the eBPF pod has to run as privileged. [IMPORTANT]
====
`NetworkEvents` is available as a Developer Preview.
 ====
 <br><br/>
```

Reply:
Or, minimally, if the admonition doesn't translate between the source and asciidoc, just using all caps IMPORTANT: